spring-data-microservice-transaction-sample/README.md

```diff
@@ -0,0 +1,548 @@
+# Sample application of Spring Data JDBC for ScalarDB with Microservice Transactions
+
+This tutorial describes how to create a sample Spring Boot application for microservice transactions by using Spring Data JDBC for ScalarDB with Two-phase Commit Transactions in ScalarDB.
```

Comment:
```suggestion
This tutorial describes how to create a sample Spring Boot application for microservice transactions by using Spring Data JDBC for ScalarDB.
```
The original one is logically correct, but I think it's a bit hard to follow.
Is it OK to just say it this way without losing the meaning?

Reply:
Yeah, the suggested one is simpler and better. Thanks!